cmd/alias.go

```diff
@@ -11,19 +11,62 @@ import (
 	"os"
 	"path/filepath"
 	"strings"
-	. "trellis-cli/templates"
 	"trellis-cli/trellis"
 )
 
 type AliasCommand struct {
-	UI      cli.Ui
-	flags   *flag.FlagSet
-	Trellis *trellis.Trellis
-	local   string
+	UI                cli.Ui
+	flags             *flag.FlagSet
+	Trellis           *trellis.Trellis
+	local             string
+	aliasPlaybook     PlaybookInterface
+	aliasCopyPlaybook PlaybookInterface
 }
 
+const aliasYml = `
+---
+- hosts: web:&{{ env }}
+  connection: local
+  gather_facts: false
+  tasks:
+    - file:
+        path: "{{ trellis_alias_temp_dir }}"
+        state: directory
+        mode: '0755'
+      with_dict: "{{ wordpress_sites }}"
+      run_once: true
+    - template:
+        src: "{{ trellis_alias_j2 }}"
+        dest: "{{ trellis_alias_temp_dir }}/{{ env }}.yml.part"
+        mode: '0644'
+      with_dict: "{{ wordpress_sites }}"
+      run_once: true
+`
+
+const aliasYmlJ2 = `
+@{{ env }}:
+  ssh: "{{ web_user }}@{{ ansible_host }}:{{ ansible_port | default('22') }}"
+  path: "{{ project_root | default(www_root + '/' + item.key) | regex_replace('^~\/','') }}/{{ item.current_path | default('current') }}/web/wp"
+`
+
+const aliasCopyYml = `
+---
+- hosts: web:&{{ env }}
+  connection: local
+  gather_facts: false
+  tasks:
+    - copy:
+        src: "{{ trellis_alias_combined }}"
+        dest: "{{ item.value.local_path }}/wp-cli.trellis-alias.yml"
+        mode: '0644'
+        force: yes
+        decrypt: no
+      with_dict: "{{ wordpress_sites }}"
+      run_once: true
+`
+
 func NewAliasCommand(ui cli.Ui, trellis *trellis.Trellis) *AliasCommand {
-	c := &AliasCommand{UI: ui, Trellis: trellis}
+	c := &AliasCommand{UI: ui, Trellis: trellis, aliasPlaybook: &Playbook{}, aliasCopyPlaybook: &Playbook{}}
```

Comment:
- Removed `SetFiles`
- Removed the 3rd parameter (`ui cli.Ui`) from `PlaybookRunner.Run`
Passing both during struct construction.

However, `SetRoot` has to be kept because it has to run after [`c.Trellis.LoadProject()`](https://github.com/TangRufus/trellis-cli/blob/b83f85eb5df9fdee66c0b9e4a03b1513061ab673/cmd/alias.go#L38)